### PR TITLE
OADP-563: Update Velero references to 1.9

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/OADP-563

OADP 1.1.0

Previews:  (must be connected to VPN to view)

http://file.rdu.redhat.com/~ahardin/July-2022/sbreskin/backup_and_restore/index.html#application-backup-restore-operations-overview

http://file.rdu.redhat.com/~ahardin/July-2022/sbreskin/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html

http://file.rdu.redhat.com/~ahardin/July-2022/sbreskin/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html

http://file.rdu.redhat.com/~ahardin/July-2022/sbreskin/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.html

http://file.rdu.redhat.com/~ahardin/July-2022/sbreskin/backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.html

http://file.rdu.redhat.com/~ahardin/July-2022/sbreskin/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.html

### CODE REVIEW

**Comment by RichardHoch** [comment]
LGTM

File: backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc [hunk 1]
```diff
@@ -37,4 +36,4 @@ You create a default `Secret` and then you install the Data Protection Applicati
 [role="_additional-resources"]
 == Additional resources
 
-* Overview of backup locations and snapshot locations in the link:https://{velero-domain}/docs/v1.7/locations/[Velero documentation].
+* Overview of backup locations and snapshot locations in the link:https://{velero-domain}/docs/v{velero-version}/locations/[Velero documentation].
```
Comment (jeana-redhat) [inline backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc:39]: We are now supposed to [limit additional resources text](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) to the page title in a link text, like so:
```suggestion
* link:https://{velero-domain}/docs/v{velero-version}/locations/[Velero documentation].
```
You might want to use different text for the link title than what I have in the suggestion box though :)

**Comment by ahardin-rh** [comment]
@sbeskin-redhat LGTM! I added the preview builds. Can you please confirm that this is for 4.11 and let me know if you're ready for a merge? Thanks so much!

**Review by jeana-redhat — COMMENTED** [review]
Just one thing with new Additional resources requirements. I really love this use of attributes, so slick!

File: backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc]: @jeana-redhat 
Please take a look now.

Comment (jeana-redhat) [reply backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc]: Just need to remove the `.` at then end, then LGTM

Feel free to ping me in Slack when it's time to merge (even if it's after my squad shift) so you don't have to wait on another look-over :+1:

Comment (sbeskin-redhat) [reply backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc]: Done

**Comment by bergerhoffer** [comment]
The enterprise-4.12 label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.11. And any PR going into `main` must also target the latest version branch (enterprise-4.12).

If the update in your PR does NOT apply to version 4.12 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by stevsmit** [comment]
Merge review approved. This review is considered "crucial" for the OADP release. This pull request can be merged upon OADP GA without the need for a second merge review.

**Comment by stevsmit** [comment]
Apologies for not explaining. virt/backup_restore/virt-backing-up-vms.adoc is not included in the topic map, and it should be.

**Pull request merged**